grammar/Makefile

```diff
@@ -1,4 +1,7 @@
 .PHONY: all clean
+
+CFLAGS = -O3 -march=native -pipe
```

Comment:
This does more than just add CFLAGS to the Makefile, it changes the flags passed to cc.

Either convincingly motivate these changes or drop the changes to flags passed.